<!-- -*- mode: XML -*- -->

<launch>
  <!-- TPP input source -->
  <!-- 1: /LidarDetection -->
  <!-- 2: /RadarDetection -->
  <!-- 3: /DetectedObjectArray/cam60_1 -->
  <!-- otherwise: /SensorFusion -->
  <rosparam param="input_source">1</rosparam>

  <!-- TPP input/output FPS -->
  <rosparam param="input_fps">5</rosparam>
  <rosparam param="output_fps">10</rosparam>

  <!-- rviz marker lifetime -->
  <!-- 0: Auto-assign as (1 / output_fps) -->
  <!-- other value: manually assign -->
  <rosparam param="m_lifetime_sec">0</rosparam>

  <!-- rviz markers -->
  <!-- true: show probabiliy -->
  <!-- false: not show probability -->
  <rosparam param="show_probability">true</rosparam>

  <!--<node pkg="itri_tracking_pp" type="itri_tracking_pp" name="itri_tracking_pp" cwd="node" output="screen" /> -->
  <node pkg="itri_pedcross" type="pedestrian_subcriber.py" name="pedestrian_subcriber" cwd="node" output="screen" />
  <node pkg="itri_pedcross" type="itri_pedcross" name="pedestrian_event" cwd="node" output="screen" />
</launch>
